### ISSUE TITLE
bug: Some system add prefix to models (like openwebui)

### ISSUE THREAD

**Issue by rossm-mf** [langfuse/langfuse#4504]
### Describe the bug

https://github.com/open-webui/pipelines/issues/321

If you look at those issues you can see that the some system add prefix to better organize where the model come from

However all the regex wont accept any prefix as seen here : `(?i)^(gpt-)(35|3.5)(-turbo)$`

I think we should considere to remove the `^` or add a way to mass prefix a copy of some section.. Like talking all the GPT/OpenAI and make a copy of a prefix,

### To reproduce

use openwebui version 0.3.35++ and prefix your models

### SDK and container versions

since langfuse 2.7++ seen

### Are you interested to contribute a fix for this bug?

No

**Comment by marcklingen** [comment]
Interesting suggestion! @hassiebp what do you think about this?<br><br>Pros: makes it simple to account for default permutations with prefixes

Cons: can cause regressions if the model definition now applies more broadly, e.g. when wanting to define a more specific price for fine tuned models that usually have a prefix.<br><br>fyi, we will add improved model management UI very soon which will make creating model definitions way easier

**Comment by hassiebp** [comment]
@rossm-mf Thanks for raising this idea!

For the Langfuse managed model prices we aim to be as strict as possible in matching model names returned from the LLM providers. If we decide to loosen prefix / suffixes, there might be additional logic / assumptions required on our end to provide not only a match, but a *best* match for a given model string (e.g. for anthropic sonnet, allowing all prefixes would match models across different LLM providers such as Amazon Bedrock, Google Vertex AI and Anthropic themselves). We would like to avoid that.

If you have any transformations on model names for the integrations you use, please add a custom model with its corresponding regex and prices.

**Comment by rossm-mf** [comment]
It makes sense to stay strict on the models Langfuse comes builti-n. It feels like the middle-ground would be to add more continence for "custom/clone" models.

Example:

If I link the OpenAI models with the API key. I will get around 20 ish models that are basically the same price than as the builtin one. If there would be a way to a group copy or something. 

Each project has it's own models list and I have a staging/prod env. So all my tests/fixes on staging need to be ported to the prod env.

If only we had a mass import/export to use we can streamline our work ... I am writing this but maybe there is even though I dont see it in the UI

Long story short, I agree, it's not a bug but more like an enhancement.

**Comment by marcklingen** [comment]
quick addition: there is an API for this if you want to batch fetch and create new model definitions: https://api.reference.langfuse.com/
